package.json

```diff
@@ -35,7 +35,6 @@
     "docker:build:native": "scripts/in-docker.sh yarn run hak",
     "docker:build": "scripts/in-docker.sh yarn run build",
     "docker:install": "scripts/in-docker.sh yarn install",
-    "debrepo": "scripts/mkrepo.sh",
```

Comment:
Killing this script as it was unused and didn't support multiple debs in the repo (e.g. element-nightly + element-desktop)